controllers/jobController.js

```diff
@@ -40,24 +37,15 @@ module.exports = {
 
     updateJob: (req, res) => {
         let jobId = req.params.jobId;
-        let user = req.user; //new
+        let jobParams = userController.getJobParams(req, res);
 
-        jobParams = {
-            job_title: req.body.job_title,
-            location: req.body.location,
-            salary: req.body.salary,
-            company_name: req.body.company_name,
-            description: req.body.description,
-        };
-    //  Job.findByIdAndUpdate(jobId, { $set: jobParams})
         Job.findOneAndUpdate({_id: jobId}, {$set: jobParams}, {new: true}, (err, job) => {
             if(err) {
                 req.flash('error', `There has been an error while updating the job offer: ${error.message}`);
                 console.log(`Error updating job by ID: ${error.message}`);
             } else {
-                //let user = res.locals.user;
-                req.flash('success', `The job has been successfully updated!`);
-                res.redirect(`/user/${user._id}/offers`);
+                req.flash('success', `The job offer "${job.job_title}" has been successfully updated!`);
+                res.redirect(`/user/${res.locals.user._id}/offers`);
                 console.log('job updated in MongoDB and Elasticsearch')
```

Comment:
I don't know whether you wanted to delete all console logs, so just in case here is one more

Reply:
Thank you! Then I can delete one more thingy from bug ticket. 

Lets merge these changes onto development after others are finished with their current task so that all other branches wont be affected with merge conflicts just in case...